packages/dx-react-grid-bootstrap3/src/templates/table-tree-indent.jsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+export const TableTreeIndent = ({ level }) =>
+  Array.from({ length: level })
+    .map((_, currentLevel) => (
```

Comment:
We did not use underscore before as far as I remember. Why do you introduce it here?